Yoast/yoast-components
# Styling improvements

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

* Added `!important` to `h2` overrides in the Collapsible, otherwise the styling of the `#postbox h2` in WordPress will not be changed - these can be seen in the components in the metabox.
* Added `hasPadding` to the Collapsible, to make the content of the collapsible to have padding and the internal items have no padding.
* Added `hasSeparator` to the Collapsible to make borders around the entries to have them visually separated

## Test instructions

This PR can be tested by following these steps:

* Use the PR (https://github.com/Yoast/wordpress-seo/pull/10483) on Yoast SEO to see the changes.

Fixes #671
